plugins/modules/df_deployment.py

```diff
@@ -360,6 +360,10 @@ def __init__(self, module):
         self.delay = self._get_param('delay')
         self.timeout = self._get_param('timeout')
 
+        # Additional checks that can't be coded using the AnsibleModule rules
```

Comment:
can this be handled by two entries in `required_if`?

Reply:
That would've been my first implementation choice, but I don't think it can.
`required_id` enforce the _presence_ of one or more arguments if a condition is satisfied.
Here I want to enforce the _absence_ of several arguments if the condition is satisfied. I don't think this can be done with `required_if`
If you know a way, please let me know.